config_converter/config_mapper/config_mapper.py

```diff
@@ -30,15 +30,51 @@
 _SUPPORTED_PLUGINS = ['in_tail']
 
 
+def initialize_stats(d: config_pb2.Directive) -> dict:
```

Comment:
this should be named as an internal function too.

Reply:
Noted.